app/assets/stylesheets/application.css

```diff
@@ -1,15 +1,157 @@
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
- * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
- */
+@import url("https://fonts.googleapis.com/css2?family=Domine&display=swap");
+
```

Comment:
- [ ] Following the project requirements, you should follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:

  * Colors
  * Typography: font face, size and weight.
  * Layout: composition and space between elements.

Reply:
Thanks @medaminedev66, for the reviews. Ill do my best to best match the designe Color, Typo, and Layout